Tests/Pcap++Test/Tests/FilterTests.cpp

```diff
@@ -656,6 +656,35 @@ PTF_TEST_CASE(TestPcapFiltersOffline)
 	}
 	rawPacketVec.clear();
 
+	//------------------------------
+	// Composite filter - Proto + IP
+	//------------------------------
```

Comment:
If we make `CompositeFilter` abstract / not instantiable, we can probably use the existing test for `OrFilter` and `AndFilter` to test the new `removeFilter()` and `clearAllFilters()` methods

Reply:
I think it's addressed.

Reply:
Not fully actually. CompositeFilter is abstract now, but the test class just changed to use CompositeLogicFilter.

Reply:
Resolved e86630e